src/vs/workbench/contrib/chat/browser/chatWidget.ts

```diff
@@ -926,6 +924,13 @@ export class ChatWidget extends Disposable implements IChatWidget {
 		}));
 	}
 
+	startEditing(requestId: string): void {
+		const templateData = this.renderer.getTemplateDataForRequestId(requestId);
+		if (templateData) {
+			this.clickedRequest(templateData);
+		}
+	}
+
 	private clickedRequest(item: IChatListItemTemplate) {
 		this.templateData = item;
```

Comment:
It seems unsave to save this because how do you know that it's still valid and a different element hasn't been rendered into this template?

Reply:
The name `this.templateData` is very unclear